postgres-matrix.yml

```diff
@@ -5,4 +5,4 @@ version_matrix:
   - 2.4.0:
       postgres_versions: [ 10, 11, 12, 13, 14 ]
   - 2.5.0:
-      postgres_versions: [ 10, 11, 12, 13, 14, 15 ]
+      postgres_versions: [ 10, 11, 12, 13, 14, 15, 16 ]
```

Comment:
PG 10 is no longer supported by the community. I think it is time we remove it to prevent any possible headaches.